Firestore/core/test/firebase/firestore/core/query_listener_test.cc

```diff
@@ -142,13 +142,15 @@ TEST_F(QueryListenerTest, RaisesCollectionEvents) {
 }
 
 TEST_F(QueryListenerTest, RaisesErrorEvent) {
-  __block std::vector<Status> accum;
+  std::vector<Status> accum;
+  auto* accum_ptr = &accum;
   Query query = testutil::Query("rooms/Eros");
 
   auto listener = QueryListener::Create(
-      query, ^(const StatusOr<ViewSnapshot>& maybe_snapshot) {
-        accum.push_back(maybe_snapshot.status());
-      });
+      query, EventListener<ViewSnapshot>::Create(
+                 [&accum_ptr](const StatusOr<ViewSnapshot>& maybe_snapshot) {
```

Comment:
You could just pass `&accum` here. There's no reason to pass the pointer. (If you really needed to use a pointer for whatever reason, you could pass it by value to, rather than passing the pointer by reference as you've done here).

Reply:
Fixed.